rb_ws/src/buggy/scripts/auton/model_predictive_controller.py

```diff
@@ -673,8 +684,8 @@ def compute_trajectory(self, current_pose: Pose, trajectory: Trajectory, current
 
         # Publish error for debugging
         try:
-            reference_position = trajectory.get_position_by_index(
-                self.current_traj_index
+            reference_position = trajectory.get_position_by_distance(
```

Comment:
What was the reason for this change?

Reply:
the reference position should be postion at the current index + lookahead distance. Since lookahead is included in the distance variable, but not the index, this is more accurate